core/XH.js

```diff
@@ -245,7 +272,13 @@ class XHClass {
     //---------------------------
     /**
      * Show a Toast.
-     * @param {Object} config - see ToastModel for available options.
+     *
+     * @param {Object} config - options for toast instance.
+     * @param {string} message - the message to show in the toast.
```

Comment:
Shouldn't these be `config.message`, `config.icon`, etc.?

Reply:
Yes, great catch